test/test-pipe-bind-error.c

```diff
@@ -137,3 +137,23 @@ TEST_IMPL(pipe_listen_without_bind) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+TEST_IMPL(pipe_bind_or_listen_error_after_close) {
+  uv_pipe_t server;
+  int r;
+
+  r = uv_pipe_init(uv_default_loop(), &server, 0);
+  uv_close((uv_handle_t *)&server, NULL);
+
+  ASSERT(r == 0);
+  r = uv_pipe_bind(&server, TEST_PIPENAME);
+  ASSERT(r == UV_EINVAL);
+
+  r = uv_listen((uv_stream_t*)&server, SOMAXCONN, NULL);
+  ASSERT(r == UV_EINVAL);
+
+  uv_run(uv_default_loop(), UV_RUN_DEFAULT);
+
+  MAKE_VALGRIND_HAPPY();
+  return 0;
+}
```

Comment:
Please add a newline at the end of the file.

Reply:
Thanks, done.